roles/example-cnf-validate/tasks/pod-delete.yaml

```diff
@@ -49,7 +50,9 @@
 
 - name: set new testpmd node name
   set_fact:
-    testpmd_new_node: "{{ cnf_new_pods.resources[0].spec.nodeName }}"
+    testpmd_new_node: "{{ item.spec.nodeName }}"
+  with_items: "{{ cnf_new_pods.resources }}"
```

Comment:
nit: use loop instead https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_loops.html#with-items

Reply:
As always, thanks for your good advice.